core-web/libs/contenttype-fields/src/lib/fields/binary-field/store/binary-field.store.spec.ts

```diff
@@ -101,6 +103,18 @@ describe('DotBinaryFieldStore', () => {
 
             store.vm$.subscribe((state) => expect(state.status).toBe(BINARY_FIELD_STATUS.PREVIEW));
         });
+
+        it('should set DialogOpen', () => {
+            store.setDialogOpen(true);
+
+            store.vm$.subscribe((state) => expect(state.dialogOpen).toBe(true));
+        });
+
+        it('should set DropZoneActive', () => {
+            store.setDropZoneActive(true);
+
+            store.vm$.subscribe((state) => expect(state.dropZoneActive).toBe(true));
+        });
```

Comment:
all the tests that have `expect` inside a `subscribe` needs to call `done` after the `expect`. Otherwise those test could not work

Reply:
Missed that when testing.